### PR TITLE
Fix broken comment in the document

### PR DESCRIPTION
**Pull request by upsuper**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -117,13 +117,8 @@
...
 //! pub extern "system" fn Java_HelloWorld_hello(env: JNIEnv,
-//! // this is the class that owns
-//! our
-//! // static method. Not going to
-//! be
-//! // used, but still needs to
-//! have an
-//!                                              // argument slot
+//! // this is the class that owns our static method. Not going to be used, but
+//! // still needs to have an argument slot.
```
Comment (dmitry-timofeev) [inline src/lib.rs]: I'd explain _why_ it must be here:
`but still must be present to match an expected signature of a static native method.`

**Comment by jechase** [comment]
Could you go ahead and turn wrap_comments off in rustfmt.toml while you're at it? It's likely the culprit here.

**Comment by upsuper** [comment]
I guess it's more likely a bug of rustfmt which changes the meaning of the code.

**Comment by jechase** [comment]
Well, more that rustfmt doesn't really know about the content of the
comments, so it just wraps wherever it wants to. I suppose they could parse
the doc comments in the same way that rustdoc does, but it doesn't
currently.

**Pull request merged**